fs.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs')
+const get = (err, Name)=>{
+  Name.forEach((e) => {
+    let names = " ";
+    if(e !== 'cw') {
+      names = names + e;
+   console.log(names);
+    } else {
+      names = names;
```

Comment:
why do you need a else statement?

Reply:
I thought if I use if then I have to put an else statement. I guess that's not the case.

Thanks, man!!!